pkg/component/operator/document/v0/transformer/markdowntransformer.go

```diff
@@ -114,7 +114,8 @@ func (t *pdfToMarkdownTransformer) transform() (converterOutput, error) {
 		errChan <- nil
 	}()
 
-	outputBytes, err := cmdRunner.Output()
+	outputBytes, err := cmdRunner.CombinedOutput()
```

Comment:
@donch1989 I brought back the combined output, see PR description and the [QA](https://github.com/instill-ai/pipeline-backend/pull/1032#issuecomment-2996155182) for the rationale.